src/course/infrastructure/repository/course-mock-repository.ts

```diff
@@ -0,0 +1,89 @@
+import { CourseRepository } from '../../domain/repository/course-repository';
+import { Course } from '../../domain/course';
+import { CourseId } from '../../domain/course-id';
+import { CourseFilters } from 'course/domain/course-filters';
```

Comment:
Import path for CourseFilters is incorrect ('course/domain/course-filters'), causing module resolution errors.

**Sugerencia:**
```typescript
Change to the correct relative import, e.g.:
import { CourseFilters } from '../../domain/course-filters';
```